transformers/component_tests/queries/urn_bid_events_test.go

```diff
@@ -0,0 +1,85 @@
+package queries
+
+import (
+	"math/rand"
+	"strconv"
+
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/component_tests/queries/test_helpers"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/flip"
+	storageHelpers "github.com/makerdao/vdb-mcd-transformers/transformers/storage/test_helpers"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("urn bid events query", func() {
+	var (
+		usrOne, usrTwo     string
+		bidOneID, bidTwoID int
+		flipAddress        = test_data.EthFlipAddress()
+	)
+
+	BeforeEach(func() {
+		test_config.CleanTestDB(db)
+		timestamp := int64(rand.Int31())
+		blockNumber := rand.Int()
+		headerOneID := storageHelpers.CreateHeader(timestamp, blockNumber, db).Id
+		headerTwoID := storageHelpers.CreateHeader(timestamp+1, blockNumber+1, db).Id
+
+		bidOneID = rand.Int()
+		bidTwoID = rand.Int()
+		flipIlkMetadata := types.GetValueMetadata(storage.Ilk, nil, types.Bytes32)
+		bidKeys := map[types.Key]string{constants.BidId: strconv.Itoa(bidOneID)}
+		flipBidUsrMetadata := types.GetValueMetadata(storage.BidUsr, bidKeys, types.Address)
+
+		diffID := storageHelpers.CreateFakeDiffRecord(db)
+		usrOne = "0x" + test_data.RandomString(40)
+		usrTwo = "0x" + test_data.RandomString(40)
+
+		repo := flip.FlipStorageRepository{ContractAddress: flipAddress}
+		repo.SetDB(db)
+
+		// insert records used in join
+		ethIlkErr := repo.Create(diffID, headerOneID, flipIlkMetadata, test_helpers.FakeIlk.Hex)
+		Expect(ethIlkErr).NotTo(HaveOccurred())
+		ethBidUsrErr := repo.Create(diffID, headerOneID, flipBidUsrMetadata, usrOne)
+		Expect(ethBidUsrErr).NotTo(HaveOccurred())
+
+		flipBidUsrMetadata.Keys[constants.BidId] = strconv.Itoa(bidTwoID)
```

Comment:
nbd but I'd be tempted to create a new metadata object for `bidTwoID` so that there's less risk of these tests failing if the order of operations are modified (e.g. if this assignment happened before the insert on line 49)